test/scss/_internal.test.scss

```diff
@@ -0,0 +1,21 @@
+$o-brand: 'internal';
+@import '../../main';
+
+@include describe('branded palettes') {
+	@include describe('internal') {
+		@include test('has shared colours') {
+			@include assert-equal(oColorsGetPaletteColor('black'), (#000000));
+			@include assert-equal(oColorsGetPaletteColor('white'), (#ffffff));
+		};
+
+		@include test('does not have masterbrand colors') {
+			@include assert-equal(oColorsGetPaletteColor('paper'), (null));
```

Comment:
Why not throw errors?

Reply:
cc @Financial-Times/origami-core

Reply:
History of this: the default return value went from `black` to `transparent` to `undefined` to `null` none of which have any reasoning at all. I'd be OK with throwing errors but it'd be a breaking change so if we're gonna do it we should create an issue with the `breaking` label.

See:
https://github.com/Financial-Times/o-colors/pull/57
https://github.com/Financial-Times/o-colors/pull/69
https://github.com/Financial-Times/o-colors/pull/87